projects/step-frontend/src/lib/modules/timeseries/components/doughnut-chart/doughnut-chart.component.ts

```diff
@@ -29,31 +29,23 @@ import { StepCommonModule } from '../../../_common/step-common.module';
 export class DoughnutChartComponent implements OnDestroy {
   settings = input<DoughnutChartSettings>({ items: [] });
   canvas = viewChild<ElementRef<HTMLCanvasElement>>('canvas');
-  // @ViewChild('canvas') private canvas!: ElementRef<HTMLCanvasElement>;
   private chart: Chart | undefined;
 
-  // ngAfterViewInit(): void {
-  //   if (this.settings) {
-  //     this.chart = this.createChart();
-  //   }
-  // }
-
   effect = effect(() => {
     let settings = this.settings();
     let canvas = this.canvas();
     if (settings && canvas) {
-      this.createChart(settings, canvas);
+      if (this.chart) {
```

Comment:
Please align changes with https://github.com/exense/step-frontend/pull/1028/files#diff-8488e7c6db503eb8a9f01577c1ec161b448ce72bbf6bda5d82b9bd548030f9d9

To avoid conflicts

Reply:
did it. in case of conflicts, you can accept my file entirely